### PR TITLE
Delete the wrong comment about CertDirectory of kubelet 

### PR DESCRIPTION
**Pull request by tnqn**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
The default value of CertDirectory was changed to `/var/lib/kubelet/pki` in https://github.com/kubernetes/kubernetes/pull/53317/commits/8c25265314f69d2618ab90f8b588897f57a45e1d.

The comment was misleading, should either be corrected or deleted.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by mattjmcnaughton — COMMENTED** [review]
Thanks for your careful eye :)

Instead of hard-coding, could the comment direct the user to where they can find the default in the code?

I worry about this falling out of date in the future.

**Comment by tnqn** [comment]
> Thanks for your careful eye :)
> 
> Instead of hard-coding, could the comment direct the user to where they can find the default in the code?
> 
> I worry about this falling out of date in the future.

Thanks @mattjmcnaughton for reviewing. Yes, I was hesitating to correct it or delete it as it might be out of data again in the future.  If changing where to find the default value instead, could that place be changed in the future too?

How about another option: deleting comment about default value? I see many other variables like `RootDirectory` don't have such comments here, at least it won't be misleading.

File: cmd/kubelet/app/options/options.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ type KubeletFlags struct {
 	config.ContainerRuntimeOptions
 
 	// certDirectory is the directory where the TLS certs are located (by
-	// default /var/run/kubernetes). If tlsCertFile and tlsPrivateKeyFile
+	// default /var/lib/kubelet/pki). If tlsCertFile and tlsPrivateKeyFile
```
Comment (mattjmcnaughton) [inline cmd/kubelet/app/options/options.go:89]: I agree with you that it probably just makes the `by default ...` part. A incorrect comment is worse than no comment haha :)

Comment (tnqn) [reply cmd/kubelet/app/options/options.go]: Deleted this comment, PTAL.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm
/retest

**Comment by tnqn** [comment]
Thanks @mattjmcnaughton @derekwaynecarr 

/test pull-kubernetes-files-remake

**Pull request merged**